code/procs/accents.dm

```diff
@@ -868,6 +921,62 @@ proc/random_accent()
 		T.update()
 	return modded
 
+/proc/germify(var/string) // pretty much the same thing as scots and tyke, but instead with some common cognates between english and german. The list is significantly smaller
+
+	var/list/phrases = list("excuse me=entschuldigung","yes sir=javol","yes ma'am=javol","you stink=du stinkst","good morning=guten morgen"
+	,"good day=guten tag", "good evening=guten abend", "good night=guten nacht","thank you=danke","thank you very much=danke schoen","I'm sorry=es tut mir leid","that's too bad=schade","no problem=kein problem"
+	,"good luck=viel glueck","bless you=gezundheit","good afternoon=guten tag")//this is horrible. I'll come up with a better system later, but I have work to do right now. This is for multi-word phrases the document can't pick up. I'm going to convert this list to a text document when I have time.
```

Comment:
This could also just be an associative list?

Reply:
okay, I fixed the stinky code I had there before. Nice and clean now!